packages/apps/plugins/plugin-observability/src/ObservabilityPlugin.tsx

```diff
@@ -159,15 +159,14 @@ export const ObservabilityPlugin = (options: {
           observability.startSpacesMetrics(client, options.namespace),
         ]);
 
-        // TODO(burdon): Remove?
-        // if (clientPlugin.provides.firstRun) {
-        //   await dispatch({
-        //     action: ObservabilityAction.SEND_EVENT,
-        //     data: {
-        //       name: 'identity.created',
-        //     },
-        //   });
-        // }
+        if (!client.halo.identity.get()) {
```

Comment:
under what circumstances would this be needed where https://github.com/dxos/dxos/pull/7277/commits/8d362975208c71462fddacf83cee4aa86a4e1fe6#diff-9e2f9960cf303a7ff57e192290d26318bb0405752c0b57b593b895e8650450c9R156 wouldn't work?

Reply:
this PR makes it such that identity isn't created during client plugin initialize anymore so this should handle the case when first landing on composer